README.md

```diff
@@ -21,7 +21,7 @@ Python linter and code formatter, written in Rust. Available on the [Visual Stud
 Ruff can be used to replace Flake8 (plus dozens of plugins), Black, isort, pyupgrade, and more,
 all while executing tens or hundreds of times faster than any individual tool.
 
-The extension ships with `ruff==0.5.7`.
+The extension ships with `ruff==0.6.1`.
```

Comment:
The version also needs to be updated in the shields at the top of README.

Reply:
Yes, sorry for the oversight. Added in 06e6c07